app/pages/sections/supplier/supplier/ui.test.js

```diff
@@ -143,13 +164,31 @@ test('should render supplier name', async (t) => {
     .expect(heading.exists).ok()
     .expect(await extractInnerText(heading)).eql(content.supplierNameHeading)
     .expect(text.exists).ok()
-    .expect(await extractInnerText(text)).eql(supplierData.name);
+    .expect(await extractInnerText(text)).eql(supplierDataFromOrdapi.name);
 });
 
-test('should render supplier address', async (t) => {
+test('should render supplier name with data from dapi when no data from orapi and supplierId provided', async (t) => {
+  nock(solutionsApiUrl)
+    .get('/api/v1/suppliers/supplier-1')
+    .reply(200, supplierDataFromDapi);
+
   await pageSetup(t, true, true);
   await t.navigateTo(pageUrl);
 
+  const heading = Selector('h3[data-test-id="supplier-name-heading"]');
+  const text = Selector('div[data-test-id="supplier-name"]');
+
+  await t
+    .expect(heading.exists).ok()
+    .expect(await extractInnerText(heading)).eql(content.supplierNameHeading)
+    .expect(text.exists).ok()
+    .expect(await extractInnerText(text)).eql(supplierDataFromDapi.name);
+});
```

Comment:
This is nice. This is something we have missing in call-off to check we show the details from one api or the other.. 
Just the api name shouldn't be Dapi here

I'll add a tech debt task to do the same for call-off